projects/js-packages/social-logos/tools/svg-to-sprite.js

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env node
+/* eslint-disable no-console */
+const svgDir = 'build/svg-clean';
+const srcSpriteDir = 'src/svg-sprite';
+const srcCssDir = 'src/css';
```

Comment:
These are now only used once, so we could probably change them to something like the following:

```
const srcSpriteExampleHTML = 'src/svg-sprite/example.html';
const srcExampleCSS = 'src/css/example.css';
```

Reply:
If you want to, feel free to amend the PR. At the moment I'm feeling lazy and not seeing either way as making enough of a difference to be worth the effort when I can't just click "commit suggestion". 😀

Reply:
67214e295f